### PR TITLE
chore(docs): update naming to match new branding initiative

### PR DESCRIPTION
**Pull request by jherey**
# Description
This PR follows the rebranding initiative by the Marketing and Docs team to change commercetools to Composable Commerce.

https://github.com/commercetools/clients-team-internal/issues/87

NB: This PR does not update the tests, as this is out of scope.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-This repository contains Postman collections for the commercetools APIs.
+This repository contains Postman collections for Composable Commerce HTTP APIs.
 
 ## How to use Postman collections?
 
-Postman collections help you explore commercetools APIs with a setup time of minutes. You can theoretically run the whole commerce journey from creating customers and products, to creating orders where these customers purchase the products.
+Postman collections help you explore Composable Commerce HTTP APIs with a setup time of minutes. You can theoretically run the whole commerce journey from creating customers and products, to creating orders where these customers purchase the products.
 
-Key benefits of using Postman collections of commercetools APIs:
+Key benefits of using Postman collections of Composable Commerce APIs:
```
Comment (melindafekete) [inline README.md]:
```suggestion
Key benefits of using the Composable Commerce Postman collections:
```

File: README.md [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
 * Works with minimum technical knowledge.
-* No coding is required, and can be used as soon as you have created a commercetools project.
+* No coding is required, and can be used as soon as you have created a Project.
 * Easily explore all the API endpoints and achieve a holistic view of how the platform and other services operate.
 
-[Get Started here](GettingStarted.md) with commercetools Postman collections.
+[Get Started here](GettingStarted.md) with Composable Commerce Postman collections.
```
Comment (melindafekete) [inline README.md]:
```suggestion
[Get Started here](GettingStarted.md) with the Composable Commerce Postman collections.
```

File: README.md [hunk 3]
```diff
@@ -1,23 +1,23 @@
...
 
 ## Further documentation
 
-To learn more about using commercetools APIs and to find complete platform documentation, please visit https://docs.commercetools.com/
+To learn more about using Composable Commerce APIs and to find complete platform documentation, please visit https://docs.commercetools.com/
```
Comment (melindafekete) [inline README.md]:
```suggestion
To learn more about using Composable Commerce APIs and to find complete API documentation, please visit https://docs.commercetools.com/
```

**Review by melindafekete — CHANGES_REQUESTED** [review]

**Review by barbara79 — APPROVED** [review]
LGTM 👍

**Comment by jherey** [comment]
@melindafekete one more review please

**Review by melindafekete — APPROVED** [review]

**Pull request merged**